pkg/commands/git_commands/stash.go

```diff
@@ -109,3 +109,11 @@ func (self *StashCommands) SaveStagedChanges(message string) error {
 
 	return nil
 }
+
+func (self *StashCommands) StashUntrackedChanges(message string) error {
+	if err := self.cmd.New("git stash -u").Run(); err != nil {
```

Comment:
Redundant `if`, you could just `return self.cmd.New()`.

I'd also suggest using the extended form of the flag, i.e. `--include-untracked`, just so it's easier to see what's going on.